tests/mask.py

```diff
@@ -37,7 +37,7 @@
 
 import sys
 import unittest
-import numpy
+import numpy as np
```

Comment:
They always matter :wink: More seriously, cleanup patches should always be distinct from functionality patches.